packages/algob/src/internal/core/config/config-validation.ts

```diff
@@ -156,7 +160,12 @@ export function getValidationErrors(config: any): CfgErrors {
 				errors.push(net, "host", host, "hostname string (eg: http://example.com)");
 			}
 			const token = hcfg.token;
-			if (typeof token === "string" && token.length < 10) {
+			if (
+				typeof token === "string" &&
+				token.length < 10 &&
+				host !== testnetURL &&
+				host !== mainnetURL
```

Comment:
a) I think you should do this validation at line 159 (check above)

b) If you're doing this, you can do something like:
```js
const allowedHosts = [ testnetURL, mainnetURL, ..];

// later
if (!allowedHosts.includes(host)) { // handle }
```

c) But why do we need this at all? Host can be multiple values (local, purestake, algoexplorerapi ..etc). Why do we need this?